tensorboard/plugins/scalar/vz_line_chart/vz-chart-helpers.ts

```diff
@@ -41,6 +41,12 @@ export interface Point {
   dataset: Plottable.Dataset;
 }
 
+export interface TooltipColumn {
```

Comment:
Considering how it reads at the call site sounds like a great heuristic. Used `evaluate`.